### DISCUSSION TITLE
1.5 display with NV3030B controller

### DISCUSSION THREAD

**Discussion by ruenes1** [Bodmer/TFT_eSPI#3130] [New processor/display support or capability request] [answered]
Hi,

is the NV3030B controller supported or planned to be supported by the library? I was thinking on this specific display. 

[https://www.waveshare.com/wiki/1.5inch_LCD_Module](https://www.waveshare.com/wiki/1.5inch_LCD_Module)

The vendor provides some coding examples, but they look quite limited in comparison with the functionalities of TFT-eSPI.

Thanks!

**Comment by Bodmer** [comment, marked as answer]
It is not currently supported. There are no plans to add support. See ReadMe for supported controllers.
